docs/migration/11.x-color.md

```diff
@@ -0,0 +1,178 @@
+# Color Tokens
+
+**Note: everything in this file is a work-in-progress and will be changed.**
+
+| **Contextual tokens**     |                    |              |                                                                                            |                                          |                                          |                                          |                                          |                                          |
+|---------------------------|--------------------|--------------|--------------------------------------------------------------------------------------------|------------------------------------------|------------------------------------------|------------------------------------------|------------------------------------------|------------------------------------------|
+| **New Token Name**        | **Old Token Name** | **Status**   | **Usage**                                                                                  | **White**                                | **Gray 10**                              | **Gray 80**                              | **Gray 90**                              | **Gray 100**                             |
```

Comment:
This looks great. Just have a few suggestions. I think it would be good to refer to these as v10 and v11 rather than new and old. I also think maybe it would be better to have v10 column first, and then v11. I feel like that's the typical order we do for other migration docs so that people can find what they are "currently" on and then see what needs to change. Also, not sure if the status column is really needed. Is that useful for designers? I think on the dev side it's not important, but if design needs it then makes sense to leave it. 
```suggestion
| **v11**        | **v10** | **Status**   | **Usage**                                                                                  | **White**                                | **Gray 10**                              | **Gray 80**                              | **Gray 90**                              | **Gray 100**                             |
```